include/cuco/sentinel.cuh

```diff
@@ -17,7 +17,7 @@
 #pragma once
 
 namespace cuco {
-namespace sentinel {
+inline namespace sentinel {
```

Comment:
Are we doing this to try and avoid breaking existing code using `cuco::sentinel::empty_key`?

Reply:
Yep. Removing the namespace entirely would also affect existing user code. It would be an easy fix though.

Reply:
I've heard that nested inline namespace can be a pain with nvcc. We can consider removing the namespace entirely for the refactor branch since we already have so many breaking API changes in there.

Reply:
We can keep the inline namespace and update the git tag first in `rapids-cmake` and then update the corresponding code across RAPIDS repos. Otherwise, this will be a breaking change and cause CI failures for `cudf`, `raft` and `cugraph`.

Once it's done. we can remove the `sentinel` namespace completely.